src/clients/api/queries/getTransactions/formatTransaction.ts

```diff
@@ -0,0 +1,20 @@
+import BigNumber from 'bignumber.js';
+import { TransactionCategory, TransactionEvent } from 'types';
+import { ITransactionResponse } from './types';
+
+const formatTransaction = ({
+  amount,
+  createdAt,
+  updatedAt,
+  category,
+  event,
+  ...rest
+}: ITransactionResponse) => ({
+  ...rest,
+  amount: new BigNumber(amount),
```

Comment:
I'd suggest adding the unit:
```suggestion
  amountWei: new BigNumber(amount),
```